elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base),
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
+  BorderRadius: pfe-var(ui--border-radius),
+  Color: pfe-var(text),
+  FontSize: pfe-var(font-size),
+  Height: 36px,
```

Comment:
PatternFly is computing this value using the following equation: `--pf-c-form-control--Height: calc(var(--pf-c-form-control--FontSize) * var(--pf-c-form-control--LineHeight) + var(--pf-c-form-control--BorderWidth) * 2 + var(--pf-c-form-control--PaddingTop) + var(--pf-c-form-control--PaddingBottom)); // Needed for various browsers that compute height of form elements differently`

That seems really logical, would it be helpful to do the same here using our variables instead?  We can also make a few new form-specific variables in the global sass!

Reply:
Bump